README.md

```diff
@@ -6,19 +6,19 @@ The friction to measuring software emissions isn't that we need to know how, it'
 
 ## Solution
 
-Therefore, the intent of this project is to build a single API codenamed *carbonQL* that you can use to measure your software emissions for every runtime environment. If your application runs on bare-metal servers in the private cloud, virtualized servers in the public cloud, mobile, desktop, and laptop end-user devices, the carbonQL API  gives you data for all of them.
+Therefore, the intent of this project is to build a SDK or a framework codenamed *carbonQL* that you can use to measure your software emissions for every runtime environment. If your application runs on bare-metal servers in the private cloud, virtualized servers in the public cloud, mobile, desktop, and laptop end-user devices, the carbonQL SDK  gives you data for all of them.
```

Comment:
"a SDK" -> "an SDK". S is pronounced as "ess".

Reply:
@Oleg-Zhymolokhov  As per latest understanding, removed all references to the word SDK and replaced it with framework